itou/approvals/management/commands/export_asp_approval_data.py

```diff
@@ -0,0 +1,87 @@
+import csv
+from io import StringIO
+from os.path import splitext
+
+import django.db.models as models
+from django.core.management.base import BaseCommand
+
+from itou.approvals.models import Approval
+
+
+CSV_SEPARATOR = ";"
+FILENAME_SUFFIX = "_updated"
+
+
+class ItouHashError(Exception):
+    pass
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--file-path",
+            dest="file_path",
+            required=True,
+            action="store",
+            help="Path of the ASP CSV file to import",
+        )
+        parser.add_argument("--wet-run", action="store_true", dest="wet_run")
+
+    def handle(self, file_path, wet_run=False, **options):
+        nb_unknown_approval_errors = 0
+        nb_unknown_user_errors = 0
+        nb_unknown_errors = 0
+        nb_ok_records = 0
+        end_dates = []
+
+        self.stdout.write("Update and export ASP data for approvals")
```

Comment:
Ça fait un peu peur alors qu'on ne met rien à jour la base de donnée 😥. Suggestion : "Enrich ASP data file with the end date of approval if it has changed".